api/bazel/repository_locations.bzl

```diff
@@ -17,11 +17,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "protoc plugin to generate polyglot message validators",
         project_url = "https://github.com/bufbuild/protoc-gen-validate",
         use_category = ["api"],
-        sha256 = "03694205be52c4753045e7ccc949064987f6355d5e1a459e07271a1d14b8e82a",
-        version = "0.9.1",
+        sha256 = "f2c0f71591052f50d25745b2bcef2c3968e70474e70e84a19443d387bc40c31d",
+        version = "0.10.1",
         urls = ["https://github.com/bufbuild/protoc-gen-validate/archive/refs/tags/v{version}.zip"],
         strip_prefix = "protoc-gen-validate-{version}",
-        release_date = "2022-12-05",
+        release_date = "2023-03-20",
```

Comment:
Looking at CI it seems that `` `` are missing.

You can add:
```
            "org_golang_x_sys",
            "org_golang_x_mod",
```
to the `implied_untracked_deps` in line 25 below, and that should take care of it.

It may be that the indirect deps will need to be added to `bazel/dependency_imports.bzl`:
```
    go_repository(
        name = "org_golang_x_sys",
        importpath = "golang.org/x/sys",
        sum = "h1:RoyzQTK76Rktm3p4xyZslc8T8I1tBz4UEjZCzeh57mM=",
        version = "v0.0.0-20220908150016-7ac13a9a928d",
        # project_url = "https://pkg.go.dev/golang.org/x/sys",
        # last_update = "2022-09-08"
        # use_category = ["api"],
        # source = "https://github.com/bufbuild/protoc-gen-validate/blob/e959d03be5b7aeeecbbc995bfd1fb7c426d1cfda/dependencies.bzl#L455-L460"
    )
    go_repository(
        name = "org_golang_x_mod",
        importpath = "golang.org/x/mod",
        sum = "h1:6zppjxzCulZykYSLyVDYbneBfbaBIQPYMevg0bEwv2s=",
        version = "v0.6.0-dev.0.20220419223038-86c51ed26bb4",
        # project_url = "https://pkg.go.dev/golang.org/x/mod",
        # last_update = "2022-04-19"
        # use_category = ["api"],
        # source = "https://github.com/bufbuild/protoc-gen-validate/blob/e959d03be5b7aeeecbbc995bfd1fb7c426d1cfda/dependencies.bzl#L430-L435"
    )
```

Reply:
Yes. Thanks. I've never seen these types of failures before, so was confused. Thanks for helping out with this.